Formula/tezos-codec.rb

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+require File.join(File.dirname(__FILE__), "tezos")
+
+class TezosCodec < Tezos
+  init
+  desc "A client to decode and encode JSON"
+
+  bottle do
+    root_url "https://github.com/serokell/tezos-packaging/releases/download/v8.1-1/"
```

Comment:
```suggestion
    root_url "https://github.com/serokell/tezos-packaging/releases/download/#{version}/"
```

Reply:
Just in case

Reply:
http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html

Reply:
:pogchamp:

Reply:
Or even fugure out some way to abstract this in tezos client if you really want

Reply:
please, no, enough ruby for today :joy: